sdk/core/Azure.Core/src/RequestFailedException.cs

```diff
@@ -105,6 +110,7 @@ public RequestFailedException(Response response)
         public RequestFailedException(Response response, Exception? innerException)
             : this(response.Status, GetRequestFailedExceptionContent(response), innerException)
         {
+            _response = response;
```

Comment:
In relation to [your comment](https://github.com/Azure/azure-sdk-for-net/pull/35716/files#r1174008132), why not buffer it here as well? Is it just that any code path going through this constructor has either buffered it already? Given it's a public constructor, it seems like a gamble to assume so.

Reply:
it is buffered via the call to GetRequestFailedExceptionContent.

Reply:
Ah, I missed that it was inlined. 😳